src/xdsl/xdsl.v6.service.js

```diff
@@ -70,6 +70,17 @@ angular.module("ovh-api-services").service("OvhApiXdslV6", function ($resource,
                 method: "POST",
                 url: "/xdsl/:xdslId/updateInvalidOrMissingRio",
                 interceptor: interceptor
+            },
+            getTasks: {
```

Comment:
Why not moving this to a dedicated `xdsl/task` folder?

Reply:
Because I just need to access to only one route.